### ISSUE TITLE
Implement handling for "event.participants.update.all" from HPB

### ISSUE THREAD

**Issue by nickvergessen** [nextcloud/talk-android#1881]
## Steps to reproduce
0. Run HPB
1. Create a conversation
2. Restrict permissions
3. Start a call on web as moderator
4. Join as a normal user in the app
5. As moderator "End meeting for everyone"

### Expected behaviour
The new event of the HPB is handled gracefully:
* All connections are terminated
* The mobile client leaves the call

```json
{"type":"event","event":{"target":"participants","type":"update","update":{"roomid":"5439271893","incall":0,"all":true}}}
```

### Actual behaviour
The event is not handled

### Ref
The spreed PR will only be merged once iOS and Android understand that message and correctly leave the meeting, but currently the "End meeting for all" does not work with more than 50 users, so this should be fixed rather soon:

* https://github.com/nextcloud/spreed/pull/6948
* https://github.com/strukturag/nextcloud-spreed-signaling/pull/208
* https://github.com/nextcloud/talk-ios/issues/744

**Comment by nickvergessen** [comment]
iOS code for inspiration: https://github.com/nextcloud/talk-ios/pull/747/files

**Comment by timkrueger** [comment]
I think this is a duplicate of #1660. It will be released with 14.0.0 and is testable with the latest [RC](https://github.com/nextcloud/talk-android/releases/tag/v14.0.0rc4).

**Comment by nickvergessen** [comment]
No, it was a follow up change we realised after #1660 got used on a big instance

**Comment by nickvergessen** [comment]
Basically something like this: https://github.com/nextcloud/talk-android/commit/f70f94e6be18c07e64bdb1b6491b48407c98bee1
Needs to happen if `all: true` is part of the HPB event and there is no participant list:

https://github.com/nextcloud/talk-android/blob/master/app/src/main/java/com/nextcloud/talk/activities/CallActivity.java#L1751-L1757

**Comment by nickvergessen** [comment]
Ref: 
https://github.com/strukturag/nextcloud-spreed-signaling/pull/208/files#diff-9752dbbe894845f1e7cc6dfe36a6775c2b0c6b5633f9700c31d413bc44914cea

So there is no `changed` array and no `users` array in this event. so I guess you also need to make sure that part is not invoked or breaking on the json parsing
